test/unit/specs/components/governance/PageGovernance.spec.js

```diff
@@ -4,7 +4,6 @@ import Vuelidate from "vuelidate"
 import PageGovernance from "renderer/components/governance/PageGovernance"
 import ModalPropose from "renderer/components/governance/ModalPropose"
 import lcdClientMock from "renderer/connectors/lcdClientMock.js"
-import { mount } from "@vue/test-utils"
```

Comment:
shouldn't we start using `test-utils` instead of `vuex-setup` magic? 🎩🐰

Reply:
Using `test-utils` and specifying each moving part was more magic than `vuex-setup` for testing the templates. I.e. the router need to be specified. I think using the `vuex-setup` is the better choice in this case. For modules I agree with just testing the functions directly.

Reply:
what would @NodeGuy do 🤣

Reply:
> what would @NodeGuy do 🤣
😢